### ISSUE TITLE
Cannot checkout review mode

### ISSUE THREAD

**Issue by abdulkareemnalband** [ankitbko/vscode-pull-request-azdo#5]
Issue Type: <b>Bug</b>

Cannot checkout review mode

git config
```sh
$> git remote  get-url origin --all
git@ssh.dev.azure.com:v3/figg/Decisions/Decisions.Web
```

config
```
        "azdoPullRequests.defaultMergeMethod": "Rebase",
        "azdoPullRequests.orgUrl": "https://dev.azure.com/figg",
        "azdoPullRequests.projectName": "Decisions",
        "azdoPullRequests.pullRequestDescription": "ask",
        "azdoPullRequests.remotes": [
            "origin"
        ]
```

<details>
<summary>
logs
</summary>
<pre>
[Info] PAT token is not provided
[Info] Looking for git repository
[Info] Git repository found, initializing review manager and pr tree view.
[Info] Review> Validate state in progress
[Info] Review> Validating state...
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch tasks/4887-upgrade-froala
[Info] Review> no matching pull request metadata found on GitHub for current branch tasks/4887-upgrade-froala
[Info] PAT token is not provided
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] Found GitHub remote
[Info] Review> Queuing additional validate state
[Info] Review> no matching pull request metadata found for current branch tasks/4887-upgrade-froala
[Info] Warning: no result data for _git/Decisions.Web for search: {"sourceRefName":"tasks/4887-upgrade-froala"}
[Info] Review> no matching pull request metadata found on GitHub for current branch tasks/4887-upgrade-froala
[Info] Review> Validating state...
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch tasks/4887-upgrade-froala
[Info] Warning: no result data for _git/Decisions.Web for search: {"sourceRefName":"tasks/4887-upgrade-froala"}
[Info] Review> no matching pull request metadata found on GitHub for current branch tasks/4887-upgrade-froala
[Info] Review> switch to Pull Request #3427 - start
[Info] PullRequestGitHelper> Branch pr/kasturi.sharma@meetingdecisions.com/3427 is from a fork. Create a remote first.
[Info] PullRequestGitHelper> create remote for https://figg.visualstudio.com/_git/decisions.web.
[Info] Review> checkout failed #{"message":"Failed to execute git","stdout":"","stderr":"ssh: connect to host figg.visualstudio.com port 22: Network is unreachable\r\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\n","exitCode":128,"gitErrorCode":"RemoteConnectionError","gitCommand":"fetch","gitArgs":["fetch","_git","bugs/5808-412-pre-conditioned-failed:pr/kasturi.sharma@meetingdecisions.com/3427","--depth=1"]}
[Info] Review> Queuing additional validate state
[Info] Review> Validating state...
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch tasks/4887-upgrade-froala
[Info] Warning: no result data for _git/Decisions.Web for search: {"sourceRefName":"tasks/4887-upgrade-froala"}
[Info] Review> no matching pull request metadata found on GitHub for current branch tasks/4887-upgrade-froala

</pre>
</details>
Extension version: 0.0.5
VS Code version: Code - Insiders 1.53.0-insider (1a6eef3170be2a11751ee50d450328d458f7bfac, 2021-01-15T05:15:40.826Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.75GB (8.27GB free)|
|Process Argv|D:\\repo\\Decisions.Web\\web.code-workspace --crash-reporter-id 690ad1b7-c84f-4e12-b9a4-ec7eef81099e|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by ankitbko** [comment]
Fixed. Released version 0.0.6. However you will need to update your vscode to `1.53.0-insider` which got released earlier today.

**Comment by abdulkareemnalband** [comment]
Thanks again
works like a charm
